rosdep/python.yaml

```diff
@@ -2720,7 +2720,15 @@ python-oyaml-pip:
 python-packaging:
   debian: [python-packaging]
   fedora: [python-packaging]
-  ubuntu: [python-packaging]
+  ubuntu:
```

Comment:
These changes are unnecessary. This approach was tested and determined not to scale well. We've left the old rules in to not break anyone using the that pattern but do not create new ones. The old ones will eventually age out when the platforms go EOL.

Reply:
Thank you! I undid those changes and updated the PR.